llvm/test/CodeGen/AVR/issue-145040.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: llc < %s -O=2 -mtriple=avr-none --mcpu=avr128db28 -verify-machineinstrs | FileCheck %s
+
+declare dso_local void @nil(i16 noundef) local_unnamed_addr addrspace(1) #1
+!3 = !{!4, !4, i64 0}
```

Comment:
These lines can be removed, we need a clear test without any interference.

Reply:
But nil() is called. Without the call the bug does not trigger.